lib/presentation/extensions/send_file_extension.dart

```diff
@@ -27,6 +28,7 @@ extension SendFileExtension on Room {
     Event? inReplyTo,
     String? editEventId,
     int? shrinkImageMaxDimension,
+    FileInfo? thumbnail,
```

Comment:
why we need this, why not do it inside?

Reply:
it will be reused when try to resend again